src/Microsoft.DncEng.PatGeneratorTool/Program.cs

```diff
@@ -130,7 +132,16 @@ private static async Task<int> Go(List<AzureDevOpsPATScopes> scopes, string[] or
                     user = $"{user.Split('\\').Last()}@microsoft.com";
                 }
 
-                credentials = new VssAadCredential(user, password);
+                var app = PublicClientApplicationBuilder
+                    .Create(AzdoClientId)
+                    .WithAuthority("https://login.microsoftonline.com/microsoft.com/")
+                    .Build();
+
+                var passwordSecureStr = new SecureString();
+                Array.ForEach(password!.ToCharArray(), passwordSecureStr.AppendChar);
```

Comment:
Why not just foreach this, rather than having to copy it into a character array just to call Array.ForEach on it.
```csharp
foreach(var c in password) passwordSecureStr.AppendChar(c);
```